packages/metal-drag-drop/src/Drag.js

```diff
@@ -723,10 +723,10 @@ class Drag extends State {
 	 */
 	toElements_(elementOrSelector) {
 		if (core.isString(elementOrSelector)) {
-			let matched = this.container.querySelectorAll(
-				`${elementOrSelector}:not([aria-grabbed="true"])`
-			);
-			return Array.prototype.slice.call(matched, 0);
+			const matched = this.container.querySelectorAll(elementOrSelector);
+			return Array.prototype
+				.slice.call(matched, 0)
+				.filter(element => element.matches(':not([aria-grabbed="true"])'));
```

Comment:
Hey @p2kmgcl, I think that it's better use `dom.match` instead of `Element.matches` for compatibility because IE11 have a partial support of this function.